AnnoyancesFilter/sections/popups.txt

```diff
@@ -143,6 +143,8 @@ brighteon.com#%#document.cookie="popupSubscription=1; path=/;";
 webmd.com##.nl-hov-mobile
 newizv.ru##.popup
 speakenglishwell.ru###overlay-sidebar-wrap
+acfun.cn##body > div.app-guide
```

Comment:
Why not `acfun.cn##.app-guide` ?
I don't see second rule on the page, pls make a screenshot.

Reply:
#appguideBox is the same element with different selector on another pages. 
![image](https://user-images.githubusercontent.com/68852737/91980446-f05b9100-ed2f-11ea-9114-a832dfda0dff.png)